docs/scripts/object_orientated_programming_poc/oop_poc_activity.py

```diff
@@ -0,0 +1,146 @@
+"""The purpose of this script is to demonstrate how to use the OOP interface for Activity.
+The following actions are shown in this script:
+1. Creating a file with an Activity
+2. Retrieve an activity by parent File
+3. Creating a second file with the same activity
+4. Modifying the activity attached to both Files
+5. Creating a table with an Activity
+"""
+import asyncio
+import os
+
+from synapseclient.models import (
+    File,
+    Activity,
+    UsedURL,
+    UsedEntity,
+    Table,
+    Column,
+    ColumnType,
+)
+import synapseclient
+
+PROJECT_ID = "syn52948289"
```

Comment:
Can you update the sharing settings on your example project so that at least DPE team members can run this script?

Reply:
I added the DPE team to this project